docs/modules/node-fetch.md

```diff
@@ -0,0 +1,58 @@
+---
+description: node-fetch was a package created for fetch to run in node (similar to cross-fetch but cross-fetch is to work equal in browser and server), it has still millions of downloads even if fetch since 18 (experimental) and 21 (stable) supports fetch API
```

Comment:
the description is used for embeds etc, its meant to be more concise. see the axios one for example:

```md
---
description: Modern alternatives to the axios package for making HTTP requests in browsers and Node.js
---
```